torchbenchmark/models/sam/install.py

```diff
@@ -0,0 +1,13 @@
+import subprocess
+import sys
+
+def pip_install_requirements():
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', '-q', '-r', 'requirements.txt'])
+
+def download_checkpoint():
+    subprocess.check_call(['wget', 'https://dl.fbaipublicfiles.com/segment_anything/sam_vit_h_4b8939.pth'])
```

Comment:
Our convention is to hide the model checkpoint files in the `torchbenchmark/models/<model>/.data` directory, to avoid commiting the checkpoint file to the code repo.